init/functions

```diff
@@ -167,7 +167,8 @@ get_prefix_from_subnet() {
     fi
 }
 
-initialize_home() {
+initialize_home() (
+    set -eo pipefail
```

Comment:
No, you can't. `set -o pipefail` works in bash only and this file needs to remain POSIX compatible (as it's sourced from other scripts, and I don't want to _require_ all of them be bash). More specifically - the wall of failed tests is caused exactly by this line.
Furthermore, you don't really rely on this in this PR. Please drop [Use set -eo pipefail in initialize_home](https://github.com/QubesOS/qubes-core-agent-linux/pull/343/commits/823b3faee365ae00bbd242ecd19f925f4cd548d0) commit from this PR (can be considered in separate cleanup PR).

Reply:
This is still the case.